# üëã About Me

Hi, I‚Äôm **Juechen (Jason) Yang**, a PhD in **Biomedical Informatics** at the University of Cincinnati with a strong background in **software engineering** and **computational biology**.  

My research focuses on developing computational methods and machine learning models for **cancer genomics, single-cell/spatial transcriptomics, and drug discovery**.

---

## üéì Education
- **PhD, Biomedical Informatics** ‚Äì University of Cincinnati (2025)  

---

## üíº Tools I developed
- **PDX database for pediatric cancers** ([pedtranscriptome.org](http://www.pedtranscriptome.org/))
- **PRADA**, a gene fusion detection tool ([GitHub](https://github.com/juechenyang/PRADA2))
- **ceSAR**, an ultra fast in silico approach for drug discovery ([sig2lead.net](http://sig2lead.net/))

---

## üõ†Ô∏è Skills
- **Programming:** Python, R, C++, Java, SAS, Bash, MySQL  
- **Data Science & Bioinformatics:** NumPy, Pandas, PyTorch, TensorFlow, Seurat, Scanpy, DESeq2, Mutect2, STAR  
- **Web Development:** R Shiny, Django, HTML, CSS, JavaScript, MySQL  
- **Cloud & Tools:** DNAnexus, Docker, AWS, Git  
- **Other:** Strong communication, teamwork, adaptability  

---

## üìö Publications (Selected)
- **Copper drives remodeling of metabolic state and progression of ccRCC**. *Cancer Discovery* (2024) [doi](https://doi.org/10.1158/2159-8290.CD-24-0187)  
- **Accelerating drug discovery and repurposing by combining transcriptional signature connectivity with docking**. *Science Advances* (2024) [doi](https://doi.org/10.1126/sciadv.adj3010)  
- **PCAT: Portal for genomic & preclinical testing data of pediatric cancer PDX models**. *Nucleic Acids Research* (2021) [doi](https://doi.org/10.1093/nar/gkaa698)  
- Full list on [Google Scholar](https://scholar.google.com/citations?user=rPGiAYoAAAAJ&hl=en)  

---

## üì´ Contact
- **Email:** juechen.yang@gmail.com  
- **Location:** Cincinnati, OH  
- **GitHub:** [juechenyang](https://github.com/juechenyang)  

---
